diagnostic.md

```diff
@@ -32,23 +36,26 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+('body').val
```

Comment:
`.val()` is both a getter and a setter but it needs to be invoked with `()` and it's typically only used on input elements.